CIScripts/BuildDockerDriver.ps1

```diff
@@ -1,23 +1,13 @@
-if ($Env:ENABLE_TRACE -eq $true) {
-    Set-PSDebug -Trace 1
-}
-
-# Refresh Path
-$Env:Path = [System.Environment]::GetEnvironmentVariable("Path", "Machine")
-
-# Stop script on error
-$ErrorActionPreference = "Stop"
-
 $Env:GOPATH=pwd
 
 New-Item -ItemType Directory ./bin
-Set-Location bin
+Push-Location bin
 
 Write-Host "Installing test runner"
 go get -u -v github.com/onsi/ginkgo/ginkgo
 
 Write-Host "Building driver"
-go build -v $Env:DRIVER_SRC_PATH
+go get -u -v $Env:DRIVER_SRC_PATH
```

Comment:
I think this actually pulls from https://github.com/codilime/contrail-windows-docker. Furthermore, I think this builds only the master branch.

I suspect that only `go build` needs to be used here...